src/components/NewForm/QuestionForm.js

```diff
@@ -42,14 +52,24 @@ const QuestionForm = (props) => {
   }
 
   function handleInputType(value, _) {
-    setShowOptions(["singleChoice", "multipleChoice"].includes(value));
+    const showOptions =
+        ["singleChoice", "multipleChoice"].includes(value) ?
```

Comment:
Why do we need to change show options for number type?

Reply:
Why are numbers not just like another input type?

Reply:
Nvm Got it through the demo video

Reply:
Probably yes. But didn't change much as this will need major refactoring in the long term.